helm_deploy/hmpps-historical-prisoner/values.yaml

```diff
@@ -23,6 +23,14 @@ generic-service:
       SecAction "id:900200,phase:1,nolog,pass,t:none,setvar:tx.allowed_methods=GET HEAD POST OPTIONS,setvar:tx.paranoia_level=3"
       # Searching for prisoners gives Invalid character in request (outside of printable chars below ascii 127) so remove
       SecRuleRemoveById 920272
+      # Allow 1c in request cookie and csrf for SQL Injection Attack Detected via libinjection
+      SecRuleUpdateTargetById 942100 "!ARGS:_csrf"
+      SecRuleUpdateTargetById 942100 "!REQUEST_COOKIES:/connect.sid/"
```

Comment:
we've got two cookies here - one for frontend components and one for the standard session